### PR TITLE
105 Env Var Creator

### PR DESCRIPTION
**Pull request by Alexicazana**
Inside the scripting tab of the settings page lives the environment variables creator, where we can create new environment variables and sync their values with the database.

### CODE REVIEW

File: UInnovateApp/src/components/settingsPage/EnvVarCreator.tsx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+		const data_accessor: DataAccessor = vmd.getRowsDataAccessor(
+			schema?.schema_name,
+			env_var_table?.table_name
+		);
+
+		const env_var_rows = await data_accessor?.fetchRows();
+		setEnvVar(env_var_rows);
+		//alternative:
+		// env_var_rows?.forEach((envVar) => {
```
Comment (tsitirides) [inline UInnovateApp/src/components/settingsPage/EnvVarCreator.tsx:104]: remove commented code dont need it

File: UInnovateApp/src/components/settingsPage/EnvVarCreator.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+			alert("Environment variable saved successfully."); // Let user know it worked yay
+		} catch (error) {
+			console.error("Error in upserting environment variable:", error);
+			alert("Failed to save environment variable."); // User sad :(
+		}
+	};
+
+	//EDIT ENV VAR FEATURE:
+	const [editMode, setEditMode] = useState(false);
+	// const [editedEnvVars, setEditedEnvVars] = useState(envVar);
```
Comment (tsitirides) [inline UInnovateApp/src/components/settingsPage/EnvVarCreator.tsx:54]: same thing here remove

**Review by tsitirides — DISMISSED** [review]
looks good, just remove the commented code, I don't see any more but may have missed some.

**Comment by Alexicazana** [comment]
Bug is resolved and i refactored the entire thing so that the data accessor function is in its own separate file, the environment variable creator and the editor are also in their own separate files; more cohesive and less coupled, way easier to debug and allows us to add additional features now that the code is cleaner and way more logical

**Review by lyubageorgieva — APPROVED** [review]
Everything works as mentioned!

**Review by lyubageorgieva — APPROVED** [review]
Looks good!

**Review by SamRfk — APPROVED** [review]
looks fine

**Review by GuardiansAscend — APPROVED** [review]
Test approval? Verified by Samy and Lyuba, but their approvals don't seem to work

**Review by lyubageorgieva — APPROVED** [review]
Looks Good!

**Pull request merged**